Error running on github workflow

In the following job, my workflow is failing. You can see the workflow result [here](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853).

```
      - name: Build AppImage
        run: |
          sudo pip3 install git+https://github.com/AppImageCrafters/appimage-builder.git
          appimage-builder --skip-tests
```

<details>

<summary>
  Workflow result
</summary>

```
Run sudo pip3 install git+https://github.com/AppImageCrafters/appimage-builder.git
Collecting git+https://github.com/AppImageCrafters/appimage-builder.git
  Cloning https://github.com/AppImageCrafters/appimage-builder.git to /tmp/pip-req-build-6e4c[9](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:10)rct
  Running command git clone --filter=blob:none --quiet https://github.com/AppImageCrafters/appimage-builder.git /tmp/pip-req-build-6e4c9rct
  Resolved https://github.com/AppImageCrafters/appimage-builder.git to commit 61c8ddde9ef44b85d7444bbe79d80b44a6a5576d
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting docker
  Downloading docker-7.0.0-py3-none-any.whl (147 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 147.6/147.6 KB 8.3 MB/s eta 0:00:00
Collecting emrichen
  Downloading emrichen-0.4.0-py3-none-any.whl (34 kB)
Collecting libconf
  Downloading libconf-2.0.1-py2.py3-none-any.whl (11 kB)
Collecting lief
  Downloading lief-0.14.1-cp3[10](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:11)-cp310-manylinux_2_28_x86_64.manylinux_2_27_x86_64.whl (2.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.7/2.7 MB 94.2 MB/s eta 0:00:00
Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from appimage-builder==1.1.1.dev61+g61c8ddd) (23.2)
Collecting pydpkg
  Downloading pydpkg-1.9.2-py3-none-any.whl (15 kB)
Collecting python-gnupg
  Downloading python_gnupg-0.5.2-py2.py3-none-any.whl (20 kB)
Requirement already satisfied: pyyaml>=5 in /usr/lib/python3/dist-packages (from appimage-builder==1.1.1.dev61+g61c8ddd) (5.4.1)
Collecting questionary
  Downloading questionary-2.0.1-py3-none-any.whl (34 kB)
Requirement already satisfied: requests in /usr/lib/python3/dist-packages (from appimage-builder==1.1.1.dev61+g61c8ddd) (2.25.1)
Collecting roam
  Downloading roam-0.3.1-py3-none-any.whl (33 kB)
Collecting ruamel.yaml
  Downloading ruamel.yaml-0.18.6-py3-none-any.whl ([11](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:12)7 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 117.8/117.8 KB 49.3 MB/s eta 0:00:00
Collecting schema
  Downloading schema-0.7.5-py2.py3-none-any.whl (17 kB)
Requirement already satisfied: urllib3 in /usr/lib/python3/dist-packages (from appimage-builder==1.1.1.dev61+g61c8ddd) (1.26.5)
Collecting requests
  Downloading requests-2.31.0-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.6/62.6 KB 33.0 MB/s eta 0:00:00
Requirement already satisfied: idna<4,>=2.5 in /usr/lib/python3/dist-packages (from requests->appimage-builder==1.1.1.dev61+g61c8ddd) (3.3)
Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python3/dist-packages (from requests->appimage-builder==1.1.1.dev61+g61c8ddd) (2020.6.20)
Collecting charset-normalizer<4,>=2
  Downloading charset_normalizer-3.3.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (142 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 142.1/142.1 KB 54.4 MB/s eta 0:00:00
Collecting jsonpath-rw~=1.4.0
  Downloading jsonpath-rw-1.4.0.tar.gz (13 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting pyaml~=21.10
  Downloading pyaml-21.10.1-py2.py3-none-any.whl (24 kB)
Collecting zstandard<0.20.0,>=0.19.0
  Downloading zstandard-0.19.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.5/2.5 MB 110.2 MB/s eta 0:00:00
Collecting arpy<3.0.0,>=2.3.0
  Downloading arpy-2.3.0.tar.gz (9.6 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting PGPy==0.6.0
  Downloading PGPy-0.6.0.tar.gz (603 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 603.0/603.0 KB 104.5 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Collecting cryptography>=39.0.1
  Downloading cryptography-42.0.3-cp39-abi3-manylinux_2_28_x86_64.whl (4.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.6/4.6 MB 115.5 MB/s eta 0:00:00
Requirement already satisfied: six<2.0.0,>=1.16.0 in /usr/lib/python3/dist-packages (from pydpkg->appimage-builder==1.1.1.dev61+g61c8ddd) (1.16.0)
Requirement already satisfied: pyasn1 in /usr/lib/python3/dist-packages (from PGPy==0.6.0->pydpkg->appimage-builder==1.1.1.dev61+g61c8ddd) (0.4.8)
Collecting prompt_toolkit<=3.0.36,>=2.0
  Downloading prompt_toolkit-3.0.36-py3-none-any.whl (386 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 386.4/386.4 KB 86.7 MB/s eta 0:00:00
Collecting ruamel.yaml.clib>=0.2.7
  Downloading ruamel.yaml.clib-0.2.8-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.manylinux_2_24_x86_64.whl (526 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 526.7/526.7 KB 96.3 MB/s eta 0:00:00
Collecting contextlib2>=0.5.5
  Downloading contextlib2-21.6.0-py2.py3-none-any.whl (13 kB)
Collecting cffi>=1.[12](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:13)
  Downloading cffi-1.16.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (443 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 443.9/443.9 KB 89.1 MB/s eta 0:00:00
Collecting decorator
  Downloading decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting ply
  Downloading ply-3.11-py2.py3-none-any.whl (49 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 49.6/49.6 KB 25.1 MB/s eta 0:00:00
Collecting wcwidth
  Downloading wcwidth-0.2.[13](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:14)-py2.py3-none-any.whl (34 kB)
Collecting pycparser
  Downloading pycparser-2.21-py2.py3-none-any.whl (118 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 118.7/118.7 KB 52.6 MB/s eta 0:00:00
Building wheels for collected packages: appimage-builder, PGPy, arpy, jsonpath-rw
  Building wheel for appimage-builder (setup.py): started
  Building wheel for appimage-builder (setup.py): finished with status 'done'
  Created wheel for appimage-builder: filename=appimage_builder-1.1.1.dev61+g61c8ddd-py3-none-any.whl size=171158 sha256=4de395bb267d67544c8107e32a6d6d08a7427c2122deedce6457390af319d21c
  Stored in directory: /tmp/pip-ephem-wheel-cache-6btqdmg7/wheels/30/bc/d7/b321f2f47e3c5b50b507a258934e09880d90ff368c6892ec24
  Building wheel for PGPy (pyproject.toml): started
  Building wheel for PGPy (pyproject.toml): finished with status 'done'
  Created wheel for PGPy: filename=PGPy-0.6.0-py3-none-any.whl size=84172 sha256=9193f02e65340d94954f7407db5993d3b1577cdad5856577b4e4d2e44f05cbcd
  Stored in directory: /root/.cache/pip/wheels/98/43/cf/8b931d43f64c6f48ae00c993740daeca9edf95074ff13faf8a
  Building wheel for arpy (setup.py): started
  Building wheel for arpy (setup.py): finished with status 'done'
  Created wheel for arpy: filename=arpy-2.3.0-py3-none-any.whl size=6197 sha256=72a16d2da0dd8580d1692af0e2efcc9f61f2869c5ec45c1720adc9455bc25d34
  Stored in directory: /root/.cache/pip/wheels/48/a6/15/ea1b47129ab3ca92d6717df9698113ed7d8c2a9ab31c137fd8
  Building wheel for jsonpath-rw (setup.py): started
  Building wheel for jsonpath-rw (setup.py): finished with status 'done'
  Created wheel for jsonpath-rw: filename=jsonpath_rw-1.4.0-py3-none-any.whl size=15[14](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:15)9 sha256=d184045bb5266869d1caa321f84e89552a3ced1c207edf49db0bbdf2bfda8d12
  Stored in directory: /root/.cache/pip/wheels/e3/76/6f/c25be6a9e6cc9985b96e8c95997d46790242c6426ef68e754c
Successfully built appimage-builder PGPy arpy jsonpath-rw
Installing collected packages: wcwidth, python-gnupg, ply, libconf, arpy, zstandard, ruamel.yaml.clib, roam, pycparser, pyaml, prompt_toolkit, lief, decorator, contextlib2, charset-normalizer, schema, ruamel.yaml, requests, questionary, jsonpath-rw, cffi, emrichen, docker, cryptography, PGPy, pydpkg, appimage-builder
  Attempting uninstall: requests
    Found existing installation: requests 2.25.1
    Not uninstalling requests at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'requests'. No files were found to uninstall.
  Attempting uninstall: cryptography
    Found existing installation: cryptography 3.4.8
    Not uninstalling cryptography at /usr/lib/python3/dist-packages, outside environment /usr
    Can't uninstall 'cryptography'. No files were found to uninstall.
Successfully installed PGPy-0.6.0 appimage-builder-1.1.1.dev61+g61c8ddd arpy-2.3.0 cffi-1.[16](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:17).0 charset-normalizer-3.3.2 contextlib2-21.6.0 cryptography-42.0.3 decorator-5.1.1 docker-7.0.0 emrichen-0.4.0 jsonpath-rw-1.4.0 libconf-2.0.1 lief-0.14.1 ply-3.11 prompt_toolkit-3.0.36 pyaml-21.10.1 pycparser-2.21 pydpkg-1.9.2 python-gnupg-0.5.2 questionary-2.0.1 requests-2.31.0 roam-0.3.1 ruamel.yaml-0.[18](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:19).6 ruamel.yaml.clib-0.2.8 schema-0.7.5 wcwidth-0.2.13 zstandard-0.[19](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:20).0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/schema.py", line 443, in validate
    return s.validate(data, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/schema.py", line 4[20](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:21), in validate
    raise SchemaMissingKeyError(message, e.format(data) if e else None)
schema.SchemaMissingKeyError: Missing key: 'comp'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/schema.py", line [40](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:41)5, in validate
    nvalue = Schema(svalue, error=e, ignore_extra_keys=i).validate(value, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/schema.py", line [44](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:45)5, in validate
    raise SchemaError([None] + x.autos, [e.format(data) if e else None] + x.errors)
schema.SchemaError: Missing key: 'comp'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/appimage-builder", line 8, in <module>
    sys.exit(__main__())
  File "/usr/local/lib/python3.10/dist-packages/appimagebuilder/__main__.py", line 44, in __main__
    schema.validate(recipe_roamer)
  File "/usr/local/lib/python3.10/dist-packages/appimagebuilder/recipe/schema.py", line [114](https://github.com/bluecherrydvr/unity/actions/runs/7949704191/job/21701168853#step:13:115), in validate
    return self.v1.validate(recipe(resolve_variables=False))
  File "/usr/local/lib/python3.10/dist-packages/schema.py", line 409, in validate
    raise SchemaError([message] + x.autos, [e.format(data) if e else None] + x.errors)
schema.SchemaError: Key 'AppImage' error:
Missing key: 'comp'
Error: Process completed with exit code 1.
```

</details>

I am not sure on what's wrong. It started to fail 3 weeks ago (at [this commit](https://github.com/bluecherrydvr/unity/commit/463e0e2974fda90625833b3888739360274a4e20)) without any changes to the workflow.

--- a/appimagebuilder/modules/prime/appimage_primer.py
+++ b/appimagebuilder/modules/prime/appimage_primer.py
@@ -90,7 +90,7 @@ class AppImagePrimer(BasePrimer):
         if self.config.comp() != "None":
             command += [ "-comp", self.config.comp()]
         else:
-            command += ["-no-compression"]
+            command += ["-noDataCompression"]
 
         self.logger.info("Creating squashfs from AppDir")
         self.logger.debug(" ".join(command))

